app/src/main/java/ch/epfl/sdp/drone3d/LoginActivity.kt

```diff
@@ -0,0 +1,49 @@
+package ch.epfl.sdp.drone3d
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.EditText
+import android.widget.ProgressBar
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+
```

Comment:
The class should be documented (as everything is private or override, only the top comment is needed)